src/lib/core/selection/pseudo-checkbox/pseudo-checkbox.ts

```diff
@@ -5,9 +5,18 @@ import {
   ElementRef,
   Renderer2,
 } from '@angular/core';
+import {IsColorable, mixinColor} from '../../common-behaviors/color';
 
 export type MdPseudoCheckboxState = 'unchecked' | 'checked' | 'indeterminate';
 
+
+// Boilerplate for applying mixins to MdChip.
+export class MdPseudoCheckboxBase {
+  constructor(public _renderer: Renderer2, public _elementRef: ElementRef) {}
```

Comment:
I had to go back to this approach because just setting the properties inside of TypeScript means that the superclass cannot set the default color on initialization (maybe a trade-off)